test_unstructured_ingest/test-ingest-local-single-file-with-pdf-infer-table-structure.sh

```diff
@@ -22,6 +22,7 @@ PYTHONPATH=. ./unstructured/ingest/main.py \
     --num-processes "$max_processes" \
     --metadata-exclude coordinates,filename,file_directory,metadata.data_source.date_processed,metadata.last_modified,metadata.detection_class_prob,metadata.parent_id,metadata.category_depth \
     --output-dir "$OUTPUT_DIR" \
+    --skip-infer-table-types "" \
```

Comment:
what is the difference in specifying empty string here vs the default None?

Reply:
add a test that validates something other than what happens by default? i.e. should require a change to expected outputs or new test?

Reply:
Currently the partition method itself has this as the default for this method: 
```python
skip_infer_table_types: List[str] = ["pdf", "jpg", "png", "xls", "xlsx"],
```
So the default for the CLI is None meaning use the default in the method. If you pass in an empty string, than a default list is used an nothing is omitted. This was actually omitting all of these any time the `--pdf-infer-table-structure` flag was set to true. I took this approach to not have to worry about keeping the default value in sync in case it ever changes on the partition method.